### DISCUSSION TITLE
Firefly 2.0.8 - cannot send IOTAs

### DISCUSSION THREAD

**Discussion by piotrpietrusewicz** [iotaledger/firefly#8201] [Help & Support]
I am not able to send IOTAs from Firefly to market address. Transaction expires after some time. Why ?

**Comment by Manubmm** [comment]
I have the same problem and I can't solve it

**Comment by begonaalvarezd** [comment]
If you are trying to send funds to an exchange, most exchanges do not allow for expiration unlock conditions, you need to send without expiration

**Reply by piotrpietrusewicz** [reply]
Are U sure ? I will try.

**Reply by piotrpietrusewicz** [reply]
It worked. Thanks a lot !!
